### PR TITLE
feat: improve buttons and colors

### PR DESCRIPTION
**Pull request by renanpontez**
CleanShot 2022-12-14 at 16 59 41@2x

### CODE REVIEW

File: src/components/atoms/Button/index.tsx [hunk 1]
```diff
@@ -11,7 +13,7 @@ export type ButtonProps = {
   fullWidth?: boolean;
```
Comment (pedrohbl03) [inline src/components/atoms/Button/index.tsx:13]: We can change fullWidth to widthType or anything, but the type this is "fit" | "full". 

The HTML class `w-${widthType}`  and widthType has a default value to prop

=)

File: src/components/atoms/Button/index.tsx [hunk 2]
```diff
@@ -28,23 +30,24 @@ const Button: React.FC<ButtonProps> = ({
...
       onClick={onClick}
       href={href}
     >
-      {isLoading && (
-        <div className="btn__loader">
-          <div className="btn__loader__spinner"></div>
-        </div>
-      )}
-      {!isLoading && (
+      {isLoading ? (
```
Comment (pedrohbl03) [inline src/components/atoms/Button/index.tsx:48]: Good code abstraction!

File: src/components/atoms/Loader/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```
Comment (pedrohbl03) [inline src/components/atoms/Loader/index.tsx:1]: I have add react-icons to project, don't have any loading svg that u like ?

**Review by pedrohbl03 — APPROVED** [review]

File: src/components/atoms/Loader/index.tsx (hunk 3 above)
Comment (renanpontez) [reply src/components/atoms/Loader/index.tsx:1]: _I've added the react-icons package to the project. Isn't there any loading svg that you liked there?_

**Pull request merged**